go/oasis-node/cmd/debug/txsource/workload/delegation.go

```diff
@@ -30,6 +31,8 @@ const (
 type delegation struct {
 	BaseWorkload
 
+	beacon beacon.Backend
```

Comment:
Could get this from `d.Consensus().Beacon()` now.

Reply:
`./delegation.go:46:29: d.BaseWorkload.Consensus().Beacon undefined (type "github.com/oasisprotocol/oasis-core/go/consensus/api".ClientBackend has no field or method Beacon)`